app/services/globus_cleanup.rb

```diff
@@ -19,5 +31,8 @@ def self.cleanup_destination(dest)
     user_id = "#{dest.user.sunet_id}@stanford.edu"
     GlobusClient.delete_access_rule(path: dest.destination_path, user_id:)
     dest.update!(deleted_at: Time.zone.now)
+    Rails.logger.info("GlobusCleanup done for globus_destination #{dest.id})")
+  rescue StandardError => e # catch any "Access rule not found" errors from Globus
+    Honeybadger.notify(e, context: { message: 'GlobusCleanup failed', globus_destination_id: dest.id })
   end
```

Comment:
move the exception handling down to the one place that is running `GlobusClient` instead of repeating in the two methods above